src/index.html

```diff
@@ -1,12 +1,303 @@
+            <a class="nav__link is-active" href=" #apple"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href=" #samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#l-and-c" data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards">
+        <div class="card cards__card" data-qa="card">
+          <img
+            class="card__product-image"
+            src="./images/imac.jpeg"
+            alt="Product Image"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="card__stars card__stars_star--4">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <span class="card__review-total">Reviews: 5</span>
+          </div>
+
+          <div class="card__price">
+            <p class="card__priceTitle">Price:</p>
+            <p class="card__priceAmount">$2,199</p>
+          </div>
+
+          <a href="#" class="card__buy" data-qa="card-hover"> buy </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            class="card__product-image"
+            src="./images/imac.jpeg"
+            alt="Product Image"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="card__stars card__stars_star--4">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <span class="card__review-total">Reviews: 5</span>
+          </div>
+
+          <div class="card__price">
+            <p class="card__priceTitle">Price:</p>
```

Comment:
```suggestion
            <p class="card__priceTitle">Price:</p>
```
use "span" tag  instead of "p" tag